Twig/Extension/SonataAdminExtension.php

```diff
@@ -87,7 +112,13 @@ protected function getTemplate(FieldDescriptionInterface $fieldDescription, $def
             $template = $environment->loadTemplate($defaultTemplate);
 
             if (null !== $this->logger) {
-                $this->logger->warning(sprintf('An error occured trying to load the template "%s" for the field "%s", the default template "%s" was used instead: "%s". ', $templateName, $fieldDescription->getFieldName(), $defaultTemplate, $e->getMessage()));
+                $this->logger->warning(sprintf(
+                    'An error occured trying to load the template "%s" for the field "%s", '.
+                    'the default template "%s" was used instead.',
+                    $templateName,
+                    $fieldDescription->getFieldName(),
+                    $defaultTemplate
+                ), array('exception' => $e));
```

Comment:
The code [was initially added](https://github.com/sonata-project/SonataAdminBundle/commit/58fa7ef20165b7a6b37c32f196d8f00afe17f15a#diff-cef2694e8930be6237942530aa1b1135R70) with an interesting comment : 

> // todo: find a better solution